docs/motivating_use_case.md

```diff
@@ -69,25 +69,25 @@ good_dependent_source(relation):
 good_tool(relation):
   lookup attestations for relation.artifact
   allow if any attestation (meets reproducible_build and
-                            attestation.details.name == relation.name)
+                            attestation.predicate.name == relation.name)
```

Comment:
Should `attestation` be `statement`?

Reply:
Oh shoot, I missed these before merging!  (Why does GitHub let you merge with open comments!)

Let me send a PR to address both of these.